.gitlab-ci.yml

```diff
@@ -55,6 +55,7 @@ compile:
     variables:
       - $CI_COMMIT_BRANCH == "main"
       - $CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+.*/
+  retry: 2
```

Comment:
looks unrelated to the PR... can you submit another one?

Reply:
Moved to #482